src/database/postgres.js

```diff
@@ -14,7 +15,7 @@ postgresModule.questions = [
 	{
 		name: 'postgres:host',
 		description: 'Host IP or address of your PostgreSQL instance',
-		default: nconf.get('postgres:host') || '127.0.0.1',
+		default: nconf.get('postgres:host') || ip.address(),
```

Comment:
The default should be to assume the database is installed on the local machine, and to use the shorthand `127.0.0.1`. If the default were the external IP of the server, and the installation was moved, the config would no longer be correct.